pkg/upbin/upbin.go

```diff
@@ -0,0 +1,398 @@
+package upbin
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"strconv"
+	"strings"
+	"time"
+	"unicode"
+
+	"github.com/go-git/go-git/v5"
+	"github.com/go-git/go-git/v5/plumbing"
+	"github.com/go-git/go-git/v5/storage/memory"
+	"github.com/manifoldco/promptui"
+	"gopkg.in/yaml.v2"
+
+	"github.com/aziontech/azion-cli/pkg/cmd/version"
+)
+
+type system string
+
+const (
+	azioncli string = "azioncli"
+
+	windows system = "windows"
```

Comment:
I don't think we currently build the binaries for windows... is this just for future proofing?

Reply:
we still don't support it but I already left the type created for future features